validator/lib/validateTileset.js

```diff
@@ -76,14 +118,35 @@ function validateTileHierarchy(root, tilesetDirectory) {
         var content = tile.content;
 
         if (!defined(tile.geometricError)) {
+            var temp_json = tile;
+            if (defined(temp_json.children)) {
+                delete temp_json.children;
+            }
+            var string_json = JSON.stringify(temp_json, undefined, 4);
+            console.log('Error in tile:');
+            console.log(string_json);
```

Comment:
Make a helper function for all these - see this comment: https://github.com/AnalyticalGraphicsInc/3d-tiles-tools/pull/108#discussion_r127732457.

Avoid using `console.log` for error messages. Everything should be contained in the message.

Creating `temp_json` isn't actually doing anything since `temp_json` and `tile` still refer to the same object. It's fine not to create the temp variable since the validation will exit right after, so modifying the json doesn't matter.